Tests/TestFuncs/TestFuncThrow/FixtureFuncThrow.swift

```diff
@@ -1,23 +1,15 @@
 import MockoloFramework
 
 
-let funcThrow2 = """
-import Foundation
-
-/// \(String.mockAnnotation)
-protocol FuncThrow {
-func update<T>(arg1: T, arg2: () throws -> T) rethrows -> T
-func update<T, U>(arg1: T, arg2: @escaping (U) throws -> ()) throws -> ((T) -> (U))
-}
-"""
-
 let funcThrow = """
 import Foundation
 
 /// \(String.mockAnnotation)
 protocol FuncThrow {
     func f1(arg: Int) throws -> String
     func f2(arg: Int) throws
+    func f3(arg: Int) throws(SomeError)
+    func f4(arg: Int) throws(SomeError) -> String
```

Comment:
Could you add other possible patterns?

- `f() throws (MyError)`
- `f() async throws(any Error)`

Reply:
Thank you!
I added these cases.: [75a0f36](https://github.com/uber/mockolo/pull/262/commits/75a0f361af0e575ae4fb2a59bb99031874a286c6)